pkg/schedulingtypes/adapters/deployment.go

```diff
@@ -108,7 +109,12 @@ func (d *FederatedDeploymentAdapter) ReconcilePlacement(fedClient fedclientset.I
 	return nil
 }
 
-func (d *FederatedDeploymentAdapter) ReconcileOverride(fedClient fedclientset.Interface, qualifiedName util.QualifiedName, result map[string]int64) error {
+func (d *FederatedDeploymentAdapter) ReconcileOverride(fedClient fedclientset.Interface, qualifiedName util.QualifiedName, untypedResult interface{}) error {
+	result, ok := untypedResult.(map[string]int64)
+	if !ok {
+		return fmt.Errorf("Wrong result type passed in FederatedDeploymentAdapter for %s", qualifiedName)
```

Comment:
Question here: For which case we will get wrong type here?

Reply:
Not necessary though, used this as safe code, the method is implementing an interface.